validator_client/src/beacon_node_fallback.rs

```diff
@@ -32,6 +38,18 @@ const UPDATE_REQUIRED_LOG_HINT: &str = "this VC or the remote BN may need updati
 /// having the correct nodes up and running prior to the start of the slot.
 const SLOT_LOOKAHEAD: Duration = Duration::from_secs(2);
 
+/// If the beacon node slot_clock is within 1 slot, this is deemed acceptable. Otherwise the node
+/// will be marked as CandidateError::TimeDiscrepancy.
+const FUTURE_SLOT_TOLERANCE: Slot = Slot::new(1);
```

Comment:
Hmm tolerance of up to 12 seconds seems a lot, what would be the impact without this?

Reply:
We have to have some amount of future slot tolerance, otherwise if a node was only very slightly ahead of the slot clock, we would refuse to use it since it would report a head of `slot + 1`.

For reference, this matches the slot tolerance that we already use in sync.